src/Kernel/Magic/DebugMagic.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Jupyter.Core;
+using Microsoft.Jupyter.Core.Protocol;
+using Microsoft.Quantum.IQSharp;
+using Microsoft.Quantum.IQSharp.Common;
+using Microsoft.Quantum.IQSharp.Jupyter;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+
+namespace Microsoft.Quantum.IQSharp.Kernel
+{
+    public class RawHtmlPayload
+    {
+        public string Value { get; set; }
+    }
+
+    public class RawHtmlEncoder : IResultEncoder
+    {
+        public string MimeType => MimeTypes.Html;
+
+        public EncodedData? Encode(object displayable) =>
+            displayable is RawHtmlPayload payload
+            ? payload.Value.ToEncodedData()
+            : (Nullable<EncodedData>)null;
+    }
```

Comment:
Just a note here, @theRoughCode has classes in the `feature/path_visualizer` branch called `DisplayableHtmlElement` and `DisplayableHtmlElementEncoder` which do basically the same thing as this. I think you'll ultimately want to use those instead of adding these new classes here.

https://github.com/microsoft/iqsharp/blob/97443e6698563a7068b899a9bf1110e65eabc4bb/src/Jupyter/Visualization/DisplayableHtmlElementEncoder.cs#L9-L41

Reply:
My PR has been merged in to master. If you pull in the changes, you should be able to use it: https://github.com/microsoft/iqsharp/blob/master/src/Jupyter/Visualization/DisplayableHtmlElementEncoder.cs

Reply:
I'll try to use those classes when I go back into the main branch to fix future bugs that will arise from testing